README.md

```diff
@@ -59,6 +59,7 @@ Just run `rails db:migrate` inside the current branch. It will roll back all pha
 
 The gem offers the following rake tasks that can be manually run according to your preferences:
 - `rails db:rollback_branches` - run it to manually rolls back phantom migrations.
+- `rails db:rollback_branches:manual` - run it to manually rolls back phantom migrations one by one.
```

Comment:
@VladislavSokov Most likely, users who want to use this task would prefer to have the automatic rollback disabled. Therefore, we should provide instructions on how to disable the automatic rollback. And, if needed, prepare the code for disabling the auto rollback.

For me, the easiest way would be to disable it using an env var, e.g. f `ACTUAL_DB_SCHEMA_AUTO_ROLLBACK_DISABLED` is defined, then it is disabled. Optionally, they could also disable it via initializer by setting the `ActualDbSchema.config[:enabled]` flag. But even if it's there, since it's not documented and no info in README, it's hard to know how to disable it, agreed?

Reply:
Please create a ticket to address this comment. Let's make a separate PR for that.